src/screens/CreatePost/index.js

```diff
@@ -106,7 +107,7 @@
   const [dataImage, setDataImage] = React.useState([]);
   const [loading, setLoading] = React.useState(false);
   const [loadingPost, setLoadingPost] = React.useState(false);
-  const [dataProfile, setDataProfile] = React.useState({});
+  const [profile] = React.useContext(Context).profile;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `profile` state is now being used instead of `dataProfile`. Make sure all instances of `dataProfile` have been replaced with `profile` to maintain consistency and avoid potential errors due to undefined variables.
```diff
+ const [profile] = React.useContext(Context).profile;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->